src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/AnalyticsService.cs

```diff
@@ -204,7 +204,7 @@ private async Task DoCaptureCall<TAnalyticsCaptureRequest>(
             Label: previewToken.Label,
             DataSetVersionId: previewToken.DataSetVersionId,
             Created: previewToken.Created,
-            Expiry: previewToken.Expiry);
+            Expiry: previewToken.Expires);
```

Comment:
I'm not familiar with this, but it looks like this is correct to stop the refactor here, and not also change it in `PreviewTokenRequest`.

This looks like it's used to record the preview token in the analytics, so I'm wondering if `Activates` should be added into that? It's not one for this PR, but it might need discussing for EES-6080.